index.js

```diff
@@ -14,51 +14,42 @@ const boxenOptions = {
     backgroundColor: "#FFFFFF"
 }
 
-
-
-const version = boxen(chalk.redBright.bold("Urltester 1.0.0"), boxenOptions);
-
-if(process.argv.length == 2 ){
-    
+const messageOne = () => {
     console.log(chalk.red("\n                Standard user manual                "));
     console.log((chalk.gray("----------------------------------------------------"))); 
     console.log(chalk.redBright("urltester filename   -"), "reports good,bad,unknown urls"); 
     console.log(chalk.redBright("urltester v |version -"), "displays tool version"); 
     console.log((chalk.redBright("----------------------------------------------------\n"))); 
-}
+}; 
```

Comment:
function to avoid help us avoid callback hell.

Reply:
Thanks for this. It makes more sense now.